phpBB/styles/prosilver/template/ucp_pm_viewfolder.html

```diff
@@ -9,19 +9,19 @@
 		<div class="pm-list-container">
 			<ul class="pm-list">
 				{% for message in messagerow %}
-					<li class="pm-item {{ message.S_ROW_COUNT is odd ? 'bg1' : 'bg2' }} {{ message.FOLDER_IMG_STYLE == 'pm_unread' ? 'pm-unread' }}">
+					<li class="pm-item {{ message.FOLDER_IMG_STYLE == 'pm_unread' ? 'pm-unread' }}">
 						<span class="pm-primary">
 
-							<a href="{message.U_VIEW_PM}" class="pm-title {{ message.FOLDER_IMG_STYLE == 'pm_unread' ? 'title-unread' : 'title-read' }}">{message.SUBJECT}</a>
+							<a href="{message.U_VIEW_PM}" class="pm-title {{ message.FOLDER_IMG_STYLE == 'pm_unread' ? 'pm-unread-pm-title' : 'title-read' }}">{message.SUBJECT}</a>
```

Comment:
This is not nessecary just use a compound selector in your css .pm-unread .pm-title { color: ...}